storage/cloud-client/hmac.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Samples to illustrate management of HMAC keys via the python client library.
+"""
+
+
+from google.cloud import storage
+
+
+def list_keys(project_id):
+    """
+    List all HMAC keys associated with the project.
+    """
+    # [START storage_list_hmac_keys]
+    # project_id = 'Your Google Cloud project ID'
+    storage_client = storage.Client(project=project_id)
```

Comment:
Does this client need to be closed? If so, we should use the with-resource statement.

Reply:
I don't believe it does; there seems to be no method to do this and I don't see any examples of it. See https://github.com/googleapis/google-cloud-python/blob/master/storage/google/cloud/storage/client.py for client source code.